extensions/sat/README.md

```diff
@@ -11,24 +11,41 @@ This document explains the fields of the Satellite Extension to a STAC Item. Sat
 The Satellite extension requires the [Instrument Fields](../../item-spec/common-metadata.md#instrument).
 
 - [Example (Landsat 8)](examples/example-landsat8.json)
+- [Example (Sentinel 1)](examples/example-sentinel1.json)
 - [JSON Schema](json-schema/schema.json)
 
 ## Item fields
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
+| sat:platform_serial_identifier | string | The International Designator, also known as COSPAR ID, and NSSDCA ID |
 | sat:orbit_state        | string        | The state of the orbit. Either `ascending` or `descending` for polar orbiting satellites, or `geostationary` for geosynchronous satellites |
+| sat:absolute_orbit     | integer       | The obsolute orbit number at the time of acquisition. |
 | sat:relative_orbit     | integer       | The relative orbit number at the time of acquisition. |
+| sat:anx_date_time     | date_time    | The Ascending Node Crossing (ANX) UTC time |
+| sat:along_track_incidence_angle | number | The along track incidence angle |
```

Comment:
It somewhat sounds like these field would be a good fit for the view extension, but not sure. Other than that I'm not exactly sure how useful these fields would be in search. Also, how commonly are these available?

Reply:
Indeed, I have not seen that a similar parameter was present in view extension. Maybe the view should also include the look angle to know if this is along or across track.